substrate/client/transaction-pool/tests/zombienet/network-specs/asset-hub-high-pool-limit-fatp.toml

```diff
@@ -0,0 +1,66 @@
+[settings]
+timeout = 1500
+
+[relaychain]
+default_image = "parity/polkadot:latest"
+default_command = "polkadot"
+chain = "rococo-local"
+
+[[relaychain.nodes]]
+name = "alice"
+rpc_port = 9944
+validator = true
+
+[[relaychain.nodes]]
+name = "bob"
+rpc_port = 9945
+validator = true
+
+[[parachains]]
+id = 2000
+chain = "asset-hub-rococo-local"
+default_command = "polkadot-parachain"
+default_image = "parity/polkadot-parachain:latest"
+cumulus_based = true
+default_args = [
+	"--force-authoring",
+	"--pool-kbytes 2048000",
+	"--pool-limit 500000",
+	"--pool-type=fork-aware",
+	"--rpc-max-connections 15000",
+	"--rpc-max-response-size 150",
+	"--rpc-max-subscriptions-per-connection=128000",
+	"--state-pruning=1024",
+	"-laura::cumulus=debug",
+	"-lbasic-authorship=debug",
+	"-lpeerset=trace",
```

Comment:
I think we can set this to debug.
trace is way too spammy for heavy tests.
I would only use `debug` for txpool, the rest can be `info`.
But I would keep them in the file, so can be easily increased.
I would apply it to all files.

Reply:
done here: [320e467](https://github.com/paritytech/polkadot-sdk/pull/7257/commits/320e467a19ef5319801de06f9b14cb14b906dcd5)